flow/Makefile

```diff
@@ -281,11 +268,11 @@ $(SDC_FILE_CLOCK_PERIOD): $(SDC_FILE)
 yosys-dependencies: $(YOSYS_DEPENDENCIES)
 
 .PHONY: do-yosys
-do-yosys: $(DONT_USE_SC_LIB)
+do-yosys: yosys-dependencies
```

Comment:
Why are you adding a dependency here?  @oharboe ok?

Reply:
Long story, but I dont want these dependencies to be bazel artifacts.

This step massages exisiting artifacts and produces e.g. merged.lib, which is quite large.

Reply:
Because most of the dependencies grouped as `yosys-dependencies` are in fact required by `do-yosys`

Reply:
> but I dont want these dependencies to be bazel artifacts.

@oharboe I'm not sure what that implies, can we add the make dependency?